.github/actions/send-webhook-notification/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Send notification through webhook'
+description: 'This action sends notifications to the given webhook url.'
+inputs:
+  webhook-url:
+    description: 'The webhook URL.'
+    required: true
+  message:
+    description: 'A message to send through webhook.'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Set build url in env
```

Comment:
```suggestion
    - name: Store build URL in an environment variable
```

Reply:
not done

Reply:
Done!

Reply:
(I somehow missed this comment, thanks for checking!)